src/components/form/file-upload.tsx

```
@@ -27,6 +28,8 @@ export enum PreviewAspectRatio {
   TALL = 'tall', // 3:4
 }
 
+const { showDestructive } = useConfirmDialog();
```

Biome (2.4.4)

[error] This hook is being called at the module level, but all hooks must be called from within a hook or component.

(lint/correctness/useHookAtTopLevel)